exercises/word-search/word_search_test.py

```diff
@@ -1,83 +1,53 @@
 import unittest
 
-from word_search import WordSearch, Point
+from word_search import search
 
 
 class WordSearchTests(unittest.TestCase):
 
-    @classmethod
-    def setUpClass(self):
-        puzzle = ('jefblpepre\n'
-                  'camdcimgtc\n'
-                  'oivokprjsm\n'
-                  'pbwasqroua\n'
-                  'rixilelhrs\n'
-                  'wolcqlirpc\n'
-                  'screeaumgr\n'
-                  'alxhpburyi\n'
-                  'jalaycalmp\n'
-                  'clojurermt')
-        self.example = WordSearch(puzzle)
+    puzzle = ['jefblpepre',
+              'camdcimgtc',
+              'oivokprjsm',
+              'pbwasqroua',
+              'rixilelhrs',
+              'wolcqlirpc',
+              'screeaumgr',
+              'alxhpburyi',
+              'jalaycalmp',
+              'clojurermt']
 
     def test_horizontal_words_left_to_right(self):
-        self.assertEqual(
-            self.example.search('clojure'),
-            (Point(0, 9), Point(6, 9))
-        )
+        self.assertEqual(search(WordSearchTests.puzzle, 'clojure'), ((9, 0), (9, 6)))
```

Comment:
Why did you reverse the `x, y` order? Normally coordinate systems use `(X, Y)` notation if I'm not mistaken.